### PR TITLE
Robo life quality improvement

### PR DESCRIPTION
**Pull request by super12pl**
## About The Pull Request
Adds sterilizer spray to robodrobe for surgery. Also adds morgue to robo on delta and pubby.

## Why It's Good For The Game
Quality of robo life improvement

## Changelog
:cl:
add: Adds sterilizer spray to robodrobe
add: Adds morgue to robotics on DeltaStation and PubbyStation
/:cl:

### CODE REVIEW

**Review by zeskorion — DISMISSED** [review]

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
AdvancedAirlockControllers fucked up and need reverting

**Comment by PowerfulBacon** [comment]
You reverted all the mapping changes except the airlocks (They are really fucked up in the mapping I think because the values of pixel_y and pixel_x are set so you might have to manually do that)

**Comment by Mick1299** [comment]
The mapping issue isn't with pixel x and y, its with airlock controllers occasionally not saving their set direction for some weird ass reason. You need to manually set all the directions for them

**Review by zeskorion — DISMISSED** [review]

**Comment by PowerfulBacon** [comment]
Check the map renders details and diffs. There is still 1 airlock that was messed up by the mapmerger

**Review by PowerfulBacon — COMMENTED** [review]
See above

**Review by PowerfulBacon — DISMISSED** [review]
Approving to remove the change request for Zesko, since he says the airlock controller should be fine, but really I wouldn't want that changed in diffs since I don't think it will be fine.

**Comment by zeskorion** [comment]
nah, looked over again, diffbot isnt very clear. gonna wait for it to be fixed

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 1]
```diff
@@ -122376,7 +122347,7 @@
 	dir = 4
 	},
 /obj/machinery/advanced_airlock_controller{
-	pixel_x = 30
+	pixel_y = 30
 	},
```
Comment (PowerfulBacon) [inline _maps/map_files/Deltastation/DeltaStation2.dmm]: This

File: _maps/map_files/PubbyStation/PubbyStation.dmm [hunk 2]
```diff
@@ -54445,7 +54395,7 @@
 	dir = 1
 	},
 /obj/machinery/advanced_airlock_controller{
-	pixel_y = 24
+	pixel_y = 26
 	},
```
Comment (PowerfulBacon) [inline _maps/map_files/PubbyStation/PubbyStation.dmm]: Not related to PR, should be removed

File: _maps/map_files/PubbyStation/PubbyStation.dmm [hunk 3]
```diff
@@ -56849,7 +56838,7 @@
...
 /obj/machinery/atmospherics/components/unary/vent_pump/high_volume,
 /obj/machinery/advanced_airlock_controller{
-	pixel_y = 24
+	pixel_y = 26
 	},
```
Comment (PowerfulBacon) [inline _maps/map_files/PubbyStation/PubbyStation.dmm]: And here

File: _maps/map_files/PubbyStation/PubbyStation.dmm [hunk 4]
```diff
@@ -59972,7 +59975,7 @@
 /area/hallway/secondary/exit/departure_lounge)
 "nGr" = (
 /obj/machinery/advanced_airlock_controller{
-	pixel_y = 24
+	pixel_y = 26
 	},
```
Comment (PowerfulBacon) [inline _maps/map_files/PubbyStation/PubbyStation.dmm]: And this

File: _maps/map_files/PubbyStation/PubbyStation.dmm [hunk 5]
```diff
@@ -62779,7 +62782,7 @@
 /area/maintenance/disposal)
 "sqW" = (
 /obj/machinery/advanced_airlock_controller{
-	pixel_y = 24
+	pixel_y = 26
 	},
```
Comment (PowerfulBacon) [inline _maps/map_files/PubbyStation/PubbyStation.dmm]: And this

File: _maps/map_files/PubbyStation/PubbyStation.dmm [hunk 6]
```diff
@@ -64786,7 +64789,8 @@
 	dir = 1
 	},
 /obj/machinery/advanced_airlock_controller{
-	pixel_y = 26
+	pixel_x = 26;
+	pixel_y = 2
 	},
```
Comment (PowerfulBacon) [inline _maps/map_files/PubbyStation/PubbyStation.dmm]: And this

File: _maps/map_files/PubbyStation/PubbyStation.dmm [hunk 7]
```diff
@@ -64906,7 +64910,8 @@
 "wKa" = (
 /obj/machinery/light/small,
 /obj/machinery/advanced_airlock_controller{
-	pixel_y = 24
+	pixel_x = 26;
+	pixel_y = 2
 	},
```
Comment (PowerfulBacon) [inline _maps/map_files/PubbyStation/PubbyStation.dmm]: And this

File: _maps/map_files/PubbyStation/PubbyStation.dmm [hunk 8]
```diff
@@ -65011,8 +65016,7 @@
 	dir = 4
 	},
 /obj/machinery/advanced_airlock_controller{
-	pixel_x = 26;
-	pixel_y = 2
+	pixel_y = 24
 	},
```
Comment (PowerfulBacon) [inline _maps/map_files/PubbyStation/PubbyStation.dmm]: And here

**Review by PowerfulBacon — COMMENTED** [review]
Bug with advanced airlock controllers cause them to break in fastdmm2, you will have to revert these

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**